BRANCHES.md

```diff
@@ -0,0 +1,55 @@
+Branches used in this repo
+==========================
+
+The document explains the branching structure that we are using in the git for VFSForGit repository as well as the forking strategy that we have adopted for contributing.

```

Comment:
"we are using in the git for VFSForGit" is a bit awkward.  How about "we are using in the VFSForGit repository"

Reply:
Yeah that will sound better.